napalm/base/__init__.py

```diff
@@ -87,12 +87,14 @@ def get_network_driver(module_name, prepend=True):
         try:
             module = importlib.import_module(module_name)
             break
-        except ImportError:
-            pass
+        except ModuleNotFoundError as e:
+            if e.name == "custom_napalm" or e.name in module_name:
```

Comment:
Isn't the first part of this unnecessary i.e. can't we just do:

```
if e.name in module_name:
```

I think we also should add a comment right before the raise e, something like:

```
# directly raise exception in case of missing dependency
raise e
```